MerlinAU.sh

```diff
@@ -2129,6 +2184,224 @@ _GetLoginCredentials_()
+    -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0' \
+    -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8' \
+    -H 'Accept-Language: en-US,en;q=0.5' \
+    -H 'Accept-Encoding: gzip, deflate' \
+    -H 'Connection: keep-alive' \
+    -H "Referer: ${NodeURLstr}/index.asp" \
+    -H 'Upgrade-Insecure-Requests: 0' \
+    --cookie '/tmp/nodecookies.txt' \
+    --max-time 2 2>&1)
+
+    if [ $? -ne 0 ]; then
+        return 2
+    fi
+
+    # Extract values using regular expressions
+    node_productid=$(echo "$htmlContent" | grep -o '"productid":"[^"]*' | sed 's/"productid":"//')
+    node_asus_device_list=$(echo "$htmlContent" | grep -o '"asus_device_list":"[^"]*' | sed 's/"asus_device_list":"//')
+    node_cfg_device_list=$(echo "$htmlContent" | grep -o '"cfg_device_list":"[^"]*' | sed 's/"cfg_device_list":"//')
+    node_firmver=$(echo "$htmlContent" | grep -o '"firmver":"[^"]*' | sed 's/"firmver":"//' | tr -d '.')
+    node_buildno=$(echo "$htmlContent" | grep -o '"buildno":"[^"]*' | sed 's/"buildno":"//')
+    node_extendno=$(echo "$htmlContent" | grep -o '"extendno":"[^"]*' | sed 's/"extendno":"//')
+    node_webs_state_flag=$(echo "$htmlContent" | grep -o '"webs_state_flag":"[^"]*' | sed 's/"webs_state_flag":"//')
+    node_webs_state_info=$(echo "$htmlContent" | grep -o '"webs_state_info":"[^"]*' | sed 's/"webs_state_info":"//')
+    node_odmpid=$(echo "$htmlContent" | grep -o '"odmpid":"[^"]*' | sed 's/"odmpid":"//')
+    node_wps_modelnum=$(echo "$htmlContent" | grep -o '"wps_modelnum":"[^"]*' | sed 's/"wps_modelnum":"//')
+    node_model=$(echo "$htmlContent" | grep -o '"model":"[^"]*' | sed 's/"model":"//')
+    node_build_name=$(echo "$htmlContent" | grep -o '"build_name":"[^"]*' | sed 's/"build_name":"//')
+    node_lan_hostname=$(echo "$htmlContent" | grep -o '"lan_hostname":"[^"]*' | sed 's/"lan_hostname":"//')
+    node_label_mac=$(echo "$htmlContent" | grep -o '"label_mac":"[^"]*' | sed 's/"label_mac":"//')
+
+    # Combine extracted information into one string
+    Node_combinedVer="$node_firmver.$node_buildno.$node_extendno"
+
+    # Perform logout request
+    curl -s -k "${NodeURLstr}/Logout.asp" \
+    -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0' \
+    -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8' \
+    -H 'Accept-Language: en-US,en;q=0.5' \
+    -H 'Accept-Encoding: gzip, deflate' \
+    -H 'Connection: keep-alive' \
+    -H "Referer: ${NodeURLstr}/Main_Login.asp" \
+    -H 'Upgrade-Insecure-Requests: 0' \
+    --cookie '/tmp/nodecookies.txt' \
+    --max-time 2 2>&1)
```

Comment:
Just found this syntax error on line #2377:

`--max-time 2 2>&1)`

The fix is:

`--max-time 2 > /tmp/logout_response.txt 2>&1`

Reply:
@Martinski4GitHub 

Fixed! Thanks for pointing it out! Of course it had to be my latest addition ;) This is why I need a second set of eyes!

Reply:
Commit: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/179/commits/b96492e619c674161301db21c0c3bbddb7bcf0bf